lib/beaker-pe/install/pe_utils.rb

```diff
@@ -146,6 +146,8 @@ def frictionless_agent_installer_cmd(host, opts, pe_version)
                 frictionless_install_opts << "#{section}:#{key}=#{value}"
               end
             end
+          elsif host[:puppet_service_debug_flag] == true
```

Comment:
Do these two cases need to be exclusive or can frictionless_install_opts take both settings?  I'm assuming the later, but don't know for certain.

Reply:
Had a SUT setup, and just tested this. Looks to still work. Command line to install:
```
bash install.bash main:candy=chocolate --puppet-service-debug
```
Install worked with the debug on, and afterwards the puppet conf was:
```
# This file can be used to override the default puppet settings.
# See the following links for more details on what settings are available:
# - https://docs.puppetlabs.com/puppet/latest/reference/config_important_settings.html
# - https://docs.puppetlabs.com/puppet/latest/reference/config_about_settings.html
# - https://docs.puppetlabs.com/puppet/latest/reference/config_file_main.html
# - https://docs.puppetlabs.com/puppet/latest/reference/configuration.html
[main]
server = cz00ahagnplx135.delivery.puppetlabs.net
certname = gcxn01tatu5cpix.delivery.puppetlabs.net
candy = chocolate
```

Reply:
Ok, then probably these can be two independent if statements.